corehq/apps/export/forms.py

```diff
@@ -501,23 +583,43 @@ def _get_datespan_filter(self):
             datespan.set_timezone(self.timezone)
             return ModifiedOnRangeFilter(gte=datespan.startdate, lt=datespan.enddate + timedelta(days=1))
 
-    def get_case_filter(self):
-        group = self._get_group()
-        if group:
-            user_ids = set(group.get_static_user_ids())
-            case_filter = [OR(
-                OwnerFilter(group._id),
-                OwnerFilter(user_ids),
-                LastModifiedByFilter(user_ids)
-            )]
+    def _get_location_filter(self, location_ids):
+        users = SQLLocation.users_at_locations_and_descendants(location_ids)
+        user_ids = [user['_id'] for user in users]
+        return OwnerFilter(user_ids)
+
+    def _get_group_independent_filters(self, mobile_user_and_group_slugs):
+        default_filters = [OwnerTypeFilter(self._get_es_user_types(mobile_user_and_group_slugs))]
+
+        location_ids = ExpandedMobileWorkerFilter.selected_location_ids(mobile_user_and_group_slugs)
+        if location_ids:
+            default_filters.append(self._get_location_filter(location_ids))
```

Comment:
Cases can also be owned by locations themselves, so I think this needs a `OwnerTypeFilter(location_ids)` somewhere too.